<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindWindowClass</class>
 <widget class="QWidget" name="FindWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>88</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FindWindow</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>291</width>
     <height>56</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLineEdit" name="searchEdit"/>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="searchButton">
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="replaceEdit"/>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="replaceButton">
      <property name="text">
       <string>Replace</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
